src/main/java/org/sagebionetworks/web/client/GlobalApplicationStateImpl.java

```diff
@@ -116,7 +116,7 @@ public static boolean isIgnoredErrorMessage(String error) {
 	public void handleUncaughtException(Throwable e) {
 		try {
 			if (!isIgnoredErrorMessage(e.getMessage())) {
-				GWT.debugger();
```

Comment:
I comment this out because it causes the page to pause during loading

Reply:
This is a global uncaught js exception handler, and something that we need to track down before merging.  Please leave in (it's there on purpose)!

Reply:
Thanks, I revert the change here.